doc/release-notes.md

```diff
@@ -49,8 +224,10 @@ See detailed [set of changes][set-of-changes].
 
 Thanks to everyone who directly contributed to this release:
 
+- AJ ONeal
 - Kittywhiskers Van Gogh
 - Konstantin Akimov
+- Odysseas Gabrielides
 - PastaPastaPasta
 - UdjinM6
```

Comment:
missing vijay: https://github.com/dashpay/dash/pulls?q=is%3Apr+is%3Amerged+author%3Avijaydasmp+milestone%3A22